sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/ctl/AsyncCtlWorkload.java

```diff
@@ -84,9 +87,19 @@
     private int queryPct;
 
     public AsyncCtlWorkload(Configuration cfg) {
+        String preferredRegionsList = cfg.getPreferredRegionsList();
+        List<String> preferredRegions = null;
+        if (StringUtils.isNotEmpty(preferredRegionsList)) {
+            String[] preferredArray = preferredRegionsList.split(",");
+            if (preferredArray != null && preferredArray.length > 0) {
+                preferredRegions = new ArrayList<>(Arrays.asList(preferredArray));
+            }
+        }
```

Comment:
please factor out this code outside of the `AsyncCtlWorkload.java` to be used by all benchmark components.